content/20.pharmaceuticals.md

```diff
@@ -48,47 +50,48 @@ Similarly, the first case of SARS was reported in November 2002 in the Guangdong
 
-Despite their phylogenetic similarity, SARS-CoV-2 emerged under very different circumstances than SARS-CoV-1 in terms of scientific knowledge about HCoV.
+Despite their phylogenetic similarity, SARS-CoV-2 emerged under very different circumstances than SARS-CoV-1 in terms of scientific knowledge about HCoVs.
 The trajectories of the pandemics associated with each of the viruses have also diverged significantly.
 By July 2003, the SARS outbreak was officially determined to be under control, with the success credited to infection management practices such as mask wearing [@doi:10.1038/nrmicro.2016.81].
 In contrast, MERS is still circulating and remains a concern; although the fatality rate is very high at almost 35%, the disease is much less easily transmitted, as its R~0~ has been estimated to be 1 [@doi:10.1093/ajcp/aqaa029].
 The low R~0~ in combination with public health practices allowed for its spread to be contained [@doi:10.1093/ajcp/aqaa029].
 Neither of these trajectories are comparable to that of SARS-CoV-2, which remains a serious threat worldwide more than a year after the first cases of COVID-19 emerged.
 
-Early results suggest that pharmaceutical interventions for COVID-19 may be more successful than efforts to develop prophylactics and therapeutics for SARS and MERS were.
+Current results suggest that pharmaceutical interventions for COVID-19 appear to be more successful than efforts to develop therapeutics for SARS and MERS were.
 Care for SARS and MERS patients prioritized supportive care and symptom management [@doi:10.1093/ajcp/aqaa029].
 To the extent that clinical treatments for SARS and MERS were explored, there is generally a lack of evidence supporting their efficacy.
 For example, Ribavirin is an antiviral that was often used in combination with corticosteroids and sometimes interferon (IFN) medications to treat SARS and MERS [@doi:10.1038/nrmicro.2016.81], but its effects have been found to be inconclusive in retrospective and _in vitro_ analyses of SARS and the SARS-CoV-1 virus, respectively [@doi:10.1038/nrmicro.2016.81].
 IFNs and Ribavirin have shown promise in _in vitro_ analyses of MERS, but their clinical effectiveness remains unknown [@doi:10.1038/nrmicro.2016.81].
 Therefore, only limited pharmaceutical advances from prior HCoV outbreaks can be adopted to COVID-19.
-Importantly, though, prior analyses of the virological and pathogenic properties of SARS-CoV-1 and MERS-CoV provide a strong foundation for the development of hypotheses about SARS-CoV-2 that have served to accelerated the development and identification of potential prophylactic and therapeutic approaches.
+Importantly, though, prior analyses of the virological and pathogenic properties of SARS-CoV-1 and MERS-CoV have provided a strong foundation for the development of hypotheses about SARS-CoV-2 that have served to accelerated the development and identification of potential therapeutic approaches.
```

Comment:
```suggestion
Importantly, though, prior analyses of the virological and pathogenic properties of SARS-CoV-1 and MERS-CoV have provided a strong foundation for the development of hypotheses about SARS-CoV-2 that have served to accelerated the development and identification of potential therapeutic approaches.
Even during the early months of the pandemic, a number of studies have emerged investigating the potential effects of drugs with mechanisms suggested based on prior understanding of coronavirus-host interactions.
Initially, these were primarily observational studies, which simply compare groups of patients who did and did not receive a treatment to determine whether it may have an effect.
However, these studies are subject to confounding, and randomized controlled trials are the standard means of assessing the effects of medications.
In randomized controlled trials, patients are prospectively and randomly assigned to treatment conditions, allowing for much stronger interpretations to be drawn; however, data from these trials takes much longer to collect.
Both have proven to be important sources of information during the COVID-19 crisis, but as more results are released from randomized controlled trials, more definitive answers are becoming available about various proposed therapeutics.
```